learning_service/src/text_preprocessing.py

```diff
@@ -25,8 +26,8 @@
 PREPROCESSOR_DATA_FILE_NAME = "preprocessor_data.joblib"
 PREPROCESSOR_LABELS_FILE_NAME = "preprocessor_labels.joblib"
 
-DATA_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "dataset")
-OUTPUT_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "output")
+DATA_PATH = os.path.join(ROOT_DIR, settings.DATASET_FOR_TRAINING_DIR)
+OUTPUT_PATH = os.path.join(ROOT_DIR, settings.OUTPUT_DIR)
```

Comment:
There is no need for joining with any kind of root dir - the variables with settings should have absolute/relative path already there.

Also, I know it's not beautiful, but using the enum values can give us at least an illusion of strong typing and highlight whenever a removed property is used or its name  is written with a typo.

```python
DATA_PATH = settings[VarNames.DATASET_FOR_TRAINING_DIR.value]
OUTPUT_PATH = settings[VarNames.OUTPUT_DIR.value]
```